/*
 * This file was automatically generated by EvoSuite
 * Thu May 29 01:40:11 GMT 2025
 */

package original;

import org.junit.Test;
import static org.junit.Assert.*;
import static org.evosuite.runtime.EvoAssertions.*;
import org.evosuite.runtime.EvoRunner;
import org.evosuite.runtime.EvoRunnerParameters;
import org.junit.runner.RunWith;
import original.Solution0475;

@RunWith(EvoRunner.class) @EvoRunnerParameters(mockJVMNonDeterminism = true, useVFS = true, useVNET = true, resetStaticState = true, separateClassLoader = true) 
public class Solution0475_ESTest extends Solution0475_ESTest_scaffolding {

  @Test(timeout = 4000)
  public void test0()  throws Throwable  {
      Solution0475 solution0475_0 = new Solution0475();
      int[] intArray0 = new int[8];
      intArray0[1] = (-1);
      intArray0[3] = (-1829);
      int int0 = solution0475_0.findRadius(intArray0, intArray0);
      assertEquals(0, int0);
  }

  @Test(timeout = 4000)
  public void test1()  throws Throwable  {
      Solution0475 solution0475_0 = new Solution0475();
      int[] intArray0 = new int[2];
      int[] intArray1 = new int[7];
      intArray1[0] = 786;
      int int0 = solution0475_0.findRadius(intArray1, intArray0);
      assertArrayEquals(new int[] {786, 0, 0, 0, 0, 0, 0}, intArray1);
      assertEquals(786, int0);
  }

  @Test(timeout = 4000)
  public void test2()  throws Throwable  {
      Solution0475 solution0475_0 = new Solution0475();
      // Undeclared exception!
      try { 
        solution0475_0.findRadius((int[]) null, (int[]) null);
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         verifyException("java.util.Arrays", e);
      }
  }
}
